app/views/books/_form.html.erb

```diff
@@ -23,10 +23,15 @@
 
   <p>
     <%= f.label :author %>
-    <%= f.text_field :species %>
+    <%= f.text_field :author %>
+  </p>
+
+  <p>
+    <%= f.label :category_id, "Categories" %><br />
+    <%= f.collection_select :category_id, Category.order(:name), :id, :name, prompt: '--select--' %>
```

Comment:
I would move `Category.order(:name)` to controller to avoid keeping logic in views when it's not necessary.